hls4ml/converters/onnx_to_hls.py

```diff
@@ -125,252 +183,95 @@ def onnx_to_hls(yamlConfig):
     layer_list = []
 
     #Extract model architecture
-    model = ModelProto()
-    with open(yamlConfig['OnnxModel'], 'rb') as fid:
-        model.ParseFromString(fid.read())
+    print('Interpreting Model ...')
+    if 'OnnxAPIModel' in config:
```

Comment:
I would prefer to just have `OnnxModel` and then figure out how to proceed based on the object that is in the config (a real model or a string that is its path on the filesystem).